server.go

```diff
@@ -1068,6 +1068,7 @@ func (s *Server) processStreamingRPC(t transport.ServerTransport, stream *transp
 		s:     stream,
 		p:     &parser{r: stream},
 		codec: s.getCodec(stream.ContentSubtype()),
+
```

Comment:
Delete this line?

Reply:
This fix is helpful for anyone using the 1.11 RC.  But not part of this PR.  Removed.